kobo/client/__init__.py

```diff
@@ -183,6 +183,22 @@ def __getattr__(self, name):
         except:
             raise AttributeError("'%s' object has no attribute '%s'" % (self.__class__.__name__, name))
 
+    @property
+    def is_logged_in(self):
+        return self._logged_in
+
+    def login(self):
+        """
+        If the hub is already logged in this method does not reattempt it.
```

Comment:
Minor: this doc string style looks odd, like the first sentence is missing.  Shouldn't it summarize what the method does first, before elaborating on one aspect of the behavior?  Like:

```python
"""Login to the hub.

If the hub is already logged in this method does not reattempt it.

Raises:
    AuthenticationError when login fails.
"""
```

Reply:
I had a summary there, but took it out because I had a thought that the method name was descriptive enough. That effectively turned the elaboration in to a single-line docstring, which was then too long for the line with the opening quotes. PEP257 allows the first line of a docstring to move down one if it's too long with the quotes in front, but in trying to pick the least funny-looking thing I think I instead picked the most funny-looking thing. Easy fix though.